packages/neon-js/src/experimental/helpers.ts

```diff
@@ -217,11 +220,20 @@ export async function addFees(
   if (config.networkFeeOverride) {
     transaction.networkFee = config.networkFeeOverride;
   } else {
-    transaction.networkFee = await calculateNetworkFee(
-      transaction,
-      config.account,
-      config
-    );
+    const rpcClient = new rpc.RPCClient(config.rpcAddress);
+    const txClone = new tx.Transaction(transaction);
+
+    if (txClone.witnesses.length < 1) {
+      txClone.addWitness(
```

Comment:
this assumes a simple signer (no multisig) which is fine. Is this constraint mentioned anywhere?

Reply:
Good point, it is not. How does the updated doc string work for you?

Reply:
LGTM.